.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: Test django-altcha-middleware
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: ['3.10', '3.11', '3.12']
+        django: ['4.2', '5.0', '5.1', '5.2']
+        include:
+          - python: '3.9'
+            django: '4.2'
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Python ${{ matrix.python }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install django~=${{ matrix.django }}.0
```

Comment:
I am wondering about your different commits on this line. What was wrong with `pip install django~=${{ matrix.django }}`? Seems odd to need to stick on the `.0`.